files/image_config/environment/motd

```diff
@@ -1,14 +1,50 @@
 You are on
-  ____   ___  _   _ _  ____
- / ___| / _ \| \ | (_)/ ___|
- \___ \| | | |  \| | | |
-  ___) | |_| | |\  | | |___
- |____/ \___/|_| \_|_|\____|
+
+
+                                    ,,,,
+                                .,,,,,,,,,,.
+                             *,,,,,,,,,,,,,,,,,.
+                         *****,,,,,,,,,,,,,,.
+                      ///*****,,,,,,,,,,,.
+                  *((////*****,,,,,,,,
+              ,#(((((////*****,,,,
+            (###(((((////*****.                                 #.
+           **###(((((/////*                                     ##.
+          *****((((((//,                                     /####(
+          *******//(/                                    ,#########
+          *******/////.            #%%#%%,            ((###########
+          ,******/////////     *%%%%%%%#####(     ,(((((###########
+           ******////////((((%%%%%%%%%%%#######(((((((((###########
+            ******///////(((((((#%%%%%%%####((((((((((((###########
+               ,*////////((((((((####%##(//(((((((((((((###########
+                 /%//////((((((((#####////(((((((((((((############
+              #&&&&&%%///((((((((#####////((((((((((((,  ##########
+           .#&&&&&&&%%%%%#(((((((#####////((((((((/      ##########
+          .***&&&&&&%%%%%%*  /(((#####////(((((.         ##########
+          *******%&&%%(         .#####/////*             ##########
+          ********//#              .##//*               /##########
+          *******///////                            .(((##########(
+          .******////////((.                     /((((((#########
+            *****////////(((((/              .((((((((((########(
+              ****///////(((((((((        ,(((((((((((((######,
+                 ,///////((((((((####/.////((((((((((((((#(
+                     *//(((((((((#####////(((((((((((((.
+                        .((((((((#####////(((((((((*
+                            /((((#####////((((((
+                               *(#####////((/
+                                   ###///,
+
+
+                          ____   ___  _   _ _  ____
+                         / ___| / _ \| \ | (_)/ ___|
+                         \___ \| | | |  \| | | |
+                          ___) | |_| | |\  | | |___
+                         |____/ \___/|_| \_|_|\____|
+
```

Comment:
I like it,but I feel it's too big. Also, I think all text underneath should be centered on the logo.

Here's more of what I'm thinking:

```
                    ,,,,,
                ,,,,,,,,,,,,.
             /***,,,,,,,,
         /(///***,,,,.
      (#(((///***,                     ,
     **##((///.                       ##
     ****//*                      *#####,
     ****/////     %%%####     (((######,
     .***/////(((#%%%%%####(((((((######,
       .**////(((((##%%#(/((((((((######,
        .&%///(((((###(//((((((((#/#####,
      %&&&&%%%#((((###(//(((((,   /#####,
     ***#&%%%#    (###(///(       /#####,
     ****///         *(,         *(#####,
     ****/////(               ((((######
      ***/////((((/       ,(((((((#####
         .////(((((### //(((((((((#*
             /(((((###(//(((((((
                *((###(//(((*
                   .##(//.
   
          ____   ___  _   _ _  ____
         / ___| / _ \| \ | (_)/ ___|
         \___ \| | | |  \| | | |
          ___) | |_| | |\  | | |___
         |____/ \___/|_| \_|_|\____|
                  
                  
-- Software for Open Networking in the Cloud --

 Unauthorized access and/or use are prohibited.
All access and/or use are subject to monitoring.

     Help:   http://azure.github.io/SONiC/
```

Reply:
I think this logo is trying to match the gradual change of the color
Actually I have a black/white version of the logo
I think that might be more suitable in the motd

Reply:
@jleveque i like your idea to center all text, looks good. Not sure about the smaller logo though, i tried that one on a real device and didn't look that good IMO.

@stcheng please share the one you have so that we can try it.

Reply:
also feel it is a little big

Reply:
I feel should be with 80-col width and 40~50 rows.

Reply:
@lguohan, my current proposal is already meeting your size criteria: 66 columns and 38 rows. Please keep in mind that in reality t looks considerably smaller than what it appears in this github ascii file. Have no issues making it smaller if you all agree, but keep in mind that the image won't look that neat anymore. Please let me know.